src/datasource.js

```diff
@@ -9,10 +9,19 @@ export class HawkularDatasource {
     this.type = instanceSettings.type;
     this.url = instanceSettings.url;
     this.name = instanceSettings.name;
-    this.tenant = instanceSettings.jsonData.tenant;
-    this.token = instanceSettings.jsonData.token;
     this.q = $q;
     this.backendSrv = backendSrv;
+    this.headers = {
+      'Content-Type': 'application/json',
+      'Hawkular-Tenant': instanceSettings.jsonData.tenant
+    };
+    if (typeof instanceSettings.basicAuth === 'string' && instanceSettings.basicAuth.length > 0) {
```

Comment:
(I did that initially because I thought it was necessary for compatibility with older versions of grafana but it turns out it's not)